eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -216,17 +216,23 @@ function Upload-Blobs
    
     LogDebug "Uploading $($PkgName)/$($DocVersion) to $($DocDest)..."
     & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/$($DocVersion)$($SASKey)" --recursive=true --cache-control "max-age=300, must-revalidate"
-
+    
     LogDebug "Handling versioning files under $($DocDest)/$($PkgName)/versioning/"
     $versionsObj = (Update-Existing-Versions -PkgName $PkgName -PkgVersion $DocVersion -DocDest $DocDest)
-
-    # we can safely assume we have AT LEAST one version here. Reason being we just completed Update-Existing-Versions
-    $latestVersion = ($versionsObj.SortedVersionArray | Select-Object -First 1).RawVersion
-
-    if ($UploadLatest -and ($latestVersion -eq $DocVersion))
+    $latestVersion = $versionsObj.LatestGAPackage 
+    if (!$latestVersion) {
+        $latestVersion = $versionsObj.LatestPreviewPackage 
+    }
+    LogDebug "Fetching the latest version. $latestVersion"
+    # Prepare the index.html which can redirect to the latest GA whenever avaiable, otherwise point to latest preview.
+    New-Item -Path $DocDir -Name "latest" -ItemType "directory"
+    New-Item -Path "$($DocDir)/latest" -Name "index.html" -ItemType "file" -Value "<meta http-equiv=`"refresh`" content=`"0; URL=$($DocDest)/$($PkgName)/$($latestVersion)/index.html`" />"
```

Comment:
Are there any downsides to using the meta refresh tag in terms of SEO or anything like that?

https://www.weboptimizers.com.au/google-warning-meta-refresh/

Reply:
I don't think we care about our search indexing for these pages so I'm not worried about that. The reason we are using meta refresh for redirecting is because that is the only way we could figure out how to do a redirect while using blob storage as a web host. Ideally we would use server redirects of some sort but we don't have that support in blob storage hosting. If you have other ideas that might work I'd be interested in other ideas.

Reply:
I don't think people will be linking directly to any `/latest/<some subfolder>/blah.html` anywhere. Given that, I can't think of any situations that this breaks.